webapp/components/Modal/ReleaseModal.spec.js

```diff
@@ -0,0 +1,150 @@
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
+import DisableModal from './ReleaseModal.vue'
```

Comment:
```suggestion
import ReleaseModal from '.'
```

Reply:
By now we have a convention that we place the component file and the test in one folder and name that folder according to the componnent name. Example:

```
RelaseModal/
  index.vue
  spec.js
```

Then you can import the component as above :point_up:

Reply:
by arrangement we have agreed on this form

RelaseModal/
  RelaseModal.vue
  RelaseModal.spec.js

Reply:
fixed